integr8ly/integreatly-operator
# [INTLY-5184] blackbox config

### Description:
Propagates status of the `selfSignedCerts` field from the RHMI installation CR to AMO in order to account for TLS cert issue causing blackbox metrics to fail.


JIRA: https://issues.redhat.com/browse/INTLY-5184

### Verification:
- Clone this repo
- Run the Integreatly Operator locally from this branch against an Openshift 4 cluster
- Wait for the installation to complete
- Change/Add `selfSignedCerts: true` to the `Spec` field in `RHMI` Installation CR 
- Verify that the change is propagated to the application monitoring CR and the blackbox-exporter configmap in the `redhat-rhmi-middleware-monitoring-operator`
- Verify blackbox metrics are functional by navigating to the Prometheus route and viewing the `probe_http_status_code` returns a non-zero status code
- Navigate to the `Endpoints Detailed` dashboard through the Grafana route and verify http_status_code shows a non-zero status code for all products
- Verify blackbox metrics are unavailable after changing selfSignedCerts to false in the `RHMI` installation CR
